i18n/en.json

```diff
@@ -1552,11 +1552,11 @@
   },
   {
     "id": "api.team.get_team_icon.filesettings_no_driver.app_error",
-    "translation": ""
+    "translation": "Invalid driver name for file settings.  Must be 'local' or 'amazons3'"
   },
   {
     "id": "api.team.get_team_icon.read_file.app_error",
-    "translation": ""
+    "translation": "Unable to read the team icon file."
```

Comment:
`Could not decode team icon metadata` (?) - basing off of other similar error messages

Reply:
I think is not the same error. I this case is about reading the bytes, and decoding is about the interpretation of the bytes in the file.